### PR TITLE
Four new conditions

### PR DESCRIPTION
**Pull request by Zack-83**
| Change        | Key                | Attribute (Source)                          | Remarks | Proof result |
| ------------- | ------------------ | ------------------------------------------- | ------- | ------------ |
| New condition | embargo            | project/dataset/preservation/embargo_period |         |              |
| New condition | existing_data      | project/dataset/existing_yesno              |         |              |
| New condition | repository_extern  | project/dataset/preservation/repository     |         |              |
| New condition | software_generated | project/dataset/usage_software              |         |              |

### CODE REVIEW

**Comment by torsten-rathmann** [comment]
This pull request contains besides new conditions a change of the <source> of an existing condition. For my review I need two hints from experts:

1.  Do catalogs using the unchanged condition still work?
2. Is the <source> in the XML the same as the "source attribute" in the RDMO readthedocs?

**Comment by torsten-rathmann** [comment]
The pull request rises a general question too. Are future catalog authors asked to use all existing condition matching their question / question sets in the catalog?

I think this isn't the case. My opinion is that they have the freedom to ignore a condition. The discussion concerning new conditions would be much more difficult if we expect catalog writers to use them, of course.

File: rdmorganiser/conditions/rdmo.xml [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+		<target_text>0</target_text>
+		<target_option/>
+	</condition>
+	<condition dc:uri="https://rdmorganiser.github.io/terms/conditions/existing_data">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>existing_data</key>
+		<dc:comment/>
+		<source dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/existing_yesno"/>
+		<relation>eq</relation>
+		<target_text>yes</target_text>
```
Comment (jochenklar) [inline rdmorganiser/conditions/rdmo.xml:18]: This should be 0 or 1, right?

File: rdmorganiser/conditions/rdmo.xml [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+		<relation>neq</relation>
+		<target_text/>
+		<target_option dc:uri="https://rdmorganiser.github.io/terms/options/preservation_repository_options/138"/>
+	</condition>
+	<condition dc:uri="https://rdmorganiser.github.io/terms/conditions/software_generated">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>software_generated</key>
+		<dc:comment/>
+		<source dc:uri="https://rdmorganiser.github.io/terms/domain/project/dataset/usage_software"/>
+		<relation>contains</relation>
```
Comment (jochenklar) [inline rdmorganiser/conditions/rdmo.xml:35]: This should be `eq`

**Review by torsten-rathmann — APPROVED** [review]

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
Please add an overview to the PR description, wich we need for the change log  later.

e.g. like https://github.com/rdmorganiser/rdmo-catalog/pull/29#issue-600992941

| Change                            | Key                                 | Attribute (Source) | Remarks     | Proof result      |
| --------------------------------- | ----------------------------------- | ---------- | ----------- | ----------------- |
| [new/edited/removed] | [key f the condition]  | [The Attribute this condition is evaluating] |  |  |

**Review by jmfrenzel — APPROVED** [review]
Seems ok to me.

**Pull request merged**